core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/MetricsService.java

```diff
@@ -171,7 +171,17 @@
     <T> Observable<DataPoint<T>> findDataPoints(MetricId<T> id, long start, long end, int limit, Order order,
             int pageSize);
 
-    Observable<Void> compressBlock(Observable<? extends MetricId<?>> metrics, long timeSlice, int pageSize);
```

Comment:
Why not return Completable here? One of its primary use cases was to address Observable<Void>. I know that we have plenty of places where we use Observable<Void>, but I would like to eventually replace them with Completable provided we only care about a terminal event which is almost always the case.